### PR TITLE
Updates Getting Started tutorial fixes

### LINKED ISSUE #15: Issues in LitmusChaos: Getting Started Tutorial
- [x] Full Stop missing in Welcome paragraph in page-1
- [x] Missing link for air-gapped environment setup in page-1
- [ ] Litmus installation link can be updated to the latest stable version in page-2
- [ ] Difference between cluster/namespace mode, and their installation steps in the docs should be referenced in page-2
- [x] Typo in "Verify Successful Registration of the Self Agent" paragraph in page-3, **automagically -> automatically**
- [x] In "Verify Successful Registration of the Self Agent" paragraph in page-3, change Portal to ChaosCenter
- [x] Typo in page-5, second paragraph, **onto -> on to**
- [x] Analytics section can be updated to Observability section in page-6, screenshots can also be updated for the same
- [ ] Done button on page-8 redirects to a link which causes a 404 error.

### PR DESCRIPTION
**Pull request by neelanjan00**
Fixes #15 

- Full Stop missing in Welcome paragraph in page-1
- Missing link for air-gapped environment setup in page-1
- Typo in "Verify Successful Registration of the Self Agent" paragraph in page-3, automagically -> automatically
- In "Verify Successful Registration of the Self Agent" paragraph in page-3, change Portal to ChaosCenter
- Typo in page-5, second paragraph, onto -> on to
- Analytics section can be updated to Observability section in page-6, screenshots can also be updated for the same

### CODE REVIEW

**Comment by neelanjan00** [comment]
@ksatchit @uditgaurav can you PTAL?

File: tutorial-getting-started/index.html [hunk 1]
```diff
@@ -102,7 +102,7 @@ <h3 is-upgraded><strong>Access and Login to ChaosCenter Dashboard</strong></h3>
 <p>You have been assigned a default project with Owner permissions </p>
 <p class="image-container"><img style="width: 624.00px" src="img/home.png"></p>
 <h3 is-upgraded><strong>Verify Successful Registration of the Self Agent </strong></h3>
-<p>Once the project is created, the cluster is automagically registered as a chaos target via installation of Litmus Chaos Agents. This is represented as &#34;Self-Agent&#34; in the Agents console of the Chaos Center Dashboard. </p>
+<p>Once the project is created, the cluster is automatically registered as a chaos target via installation of Litmus Chaos Agents. This is represented as &#34;Self-Agent&#34; in the Agents console of the ChaosCenter Dashboard. </p>
```
Comment (ksatchit) [inline tutorial-getting-started/index.html:105]: apparently, automagically is a thing in the infra world!

File: tutorial-running-chaos-workflow-on-remote-cluster/index.html [hunk 2]
```diff
@@ -308,8 +308,8 @@ <h3 is-upgraded><strong>Continue?</strong></h3>
 role.rbac.authorization.k8s.io/argo-role created
 rolebinding.rbac.authorization.k8s.io/argo-binding created
 
-💡Connecting agent to Chaos Center.
-💡Connecting agent to Chaos Center.
+💡Connecting agent to ChaosCenter.
```
Comment (ksatchit) [inline tutorial-running-chaos-workflow-on-remote-cluster/index.html:311]: Is this what we also see in litmusctl today

File: tutorial-getting-started/index.html (hunk 1 above)
Comment (neelanjan00) [reply tutorial-getting-started/index.html:105]: > apparently, automagically is a thing in the infra world!

Okay, I didn't know about this. So shall we let it be?

File: tutorial-running-chaos-workflow-on-remote-cluster/index.html (hunk 2 above)
Comment (neelanjan00) [reply tutorial-running-chaos-workflow-on-remote-cluster/index.html:311]: No, I guess we would have to update the commands as well.

File: tutorial-getting-started/index.html (hunk 1 above)
Comment (ksatchit) [reply tutorial-getting-started/index.html:105]: yes, we can

File: tutorial-running-chaos-workflow-on-remote-cluster/index.html (hunk 2 above)
Comment (neelanjan00) [reply tutorial-running-chaos-workflow-on-remote-cluster/index.html]: Updated, can you PTAL now?

File: tutorial-create-custom-workflows/index.html [hunk 3]
```diff
@@ -42,16 +42,16 @@ <h2 is-upgraded>Pre-Requisites</h2>
 <ul>
 <li>Kubernetes 1.17 or later. Recommended sizing of at least: 2 vCPUs, 8GB memory, 10GB of disk space (for persistent volume creation)</li>
 <li>Helm3 or Kubectl</li>
-<li>Access to the Internet from Cluster (For Air-Gapped Environments refer to<a href="https://docs.litmuschaos.io/docs/user-guides/air-gapped/" target="_blank"> this guide</a>)</li>
```
Comment (avaakash) [inline tutorial-create-custom-workflows/index.html:45]: Is this link mentioned someplace else? What is the reason we are not mentioning Air-Gapped Environments?

File: tutorial-running-chaos-workflow-on-remote-cluster/index.html [hunk 4]
```diff
@@ -58,28 +58,28 @@ <h2 is-upgraded>Prerequisites</h2>
...
 
       </google-codelab-step>
     
       <google-codelab-step label="Install Litmus" duration="5">
         <p>First, you need to connect to your cluster and access its shell. Click on the name of the cluster to proceed.</p>
-<p class="image-container"><img style="width: 624.00px" src="img/1875a88a92add9ec.png"></p>
+<p class="image-container"><img style="width: 624.00px" src="img/ae8c608343b3ab0e.png"></p>
 <p>Click on the <strong>connect shell</strong> button at the top, then click on the <strong>RUN IN CLOUD SHELL</strong> button.</p>
-<p class="image-container"><img style="width: 624.00px" src="img/14c1bff0958b090a.png"></p>
+<p class="image-container"><img style="width: 624.00px" src="img/402531bb0e3b84e9.png "></p>
```
Comment (avaakash) [inline tutorial-running-chaos-workflow-on-remote-cluster/index.html:82]:
```suggestion
<p class="image-container"><img style="width: 624.00px" src="img/402531bb0e3b84e9.png"></p>
```

**Review by avaakash — CHANGES_REQUESTED** [review]

File: tutorial-running-chaos-workflow-on-remote-cluster/index.html [hunk 5]
```diff
@@ -131,7 +131,7 @@ <h2 is-upgraded>What is litmusctl?</h2>
 </ul>
 <p>Note that the above are corresponding to the target cluster and not the GKE cluster where the control plane is situated. </p>
 <p>Download the litmus binary corresponding to your OS from the table below. </p>
-<aside class="warning"><p><strong>Note:</strong> If you have Litmus the Beta8(and below)  version of Chaos Center installed, you should install litmusctl v0.2.0. For higher, install litmusctl v0.3.0. This is because of some major security rules that were added in Beta9.</p>
+<aside class="warning"><p><strong>Note:</strong> If you have Litmus the Beta8(and below)  version of ChaosCenter installed, you should install litmusctl v0.2.0. For higher, install litmusctl v0.3.0. This is because of some major security rules that were added in Beta9.</p>
```
Comment (uditgaurav) [inline tutorial-running-chaos-workflow-on-remote-cluster/index.html:134]: Can we update litmus version and litmusctl version.

**Review by uditgaurav — APPROVED** [review]

**Review by avaakash — APPROVED** [review]

**Pull request merged**